cmd/install/assets/hypershift_operator.go

```diff
@@ -301,6 +303,21 @@ func (o HyperShiftOperatorDeployment) Build() *appsv1.Deployment {
 			},
 		}
 	}
+	image := o.OperatorImage
+	if o.Images != nil {
```

Comment:
You don't need nilchecking for reading from a nil map, you will get the default value (here: empty string) and ok=false if is nil

Reply:
fixed